database.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy.ext.asyncio import create_async_engine, AsyncSession
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+from settings import settings
+
+SQLALCHEMY_DATABASE_URL = settings.DATABASE_URL
+
+engine = create_async_engine(
+    SQLALCHEMY_DATABASE_URL, connect_args={"check_same_thread": False}
```

Comment:
The `connect_args` parameter with `check_same_thread` is typically used with SQLite in a synchronous context. For asynchronous engines, especially with databases other than SQLite, this argument might not be necessary or applicable. Ensure that this is intended for your specific database setup.